lesson3/index_hw2.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Document</title>
+	<link rel="stylesheet" type="text/css" href="style_hw2.css">
+</head>
+<body>
+	<div class="wrapper">
+		<header class="header">
+			<div class="logo">
+				<img src="logo.png">				
+			</div>
+			<div class="menu">
+				<ul class="menu__list">
+					<li class ="menu__items"><a class="menu__link menu__link-first" href="#">Главная</a></li>
+					<li class ="menu__items"><a class="menu__link" href="#">Товары</a></li>
+					<li class ="menu__items"><a class="menu__link" href="#">О_нас</a></li>
+					<li class ="menu__items"><a class="menu__link" href="#">Контакты</a></li>
+				</ul>
+			</div>
+		</header>
+		<div class="container">
+			<nav class="aside">
```

Comment:
good choise - nav. Use aside tag instead of div.container. Aside is not functional, it is stylistic. left-menu will be ok